holopy/core/tests/test_utils_math.py

```diff
@@ -403,9 +404,14 @@ def test_schwimmbad_multipool(self):
     @attr("fast")
     def test_MPI(self):
         if MPIPool.enabled():
+            # mpi should work
             mpi_pool = choose_pool('mpi')
             self.assertTrue(isinstance(mpi_pool, MPIPool))
+        elif find_spec('mpi4py') is None:
+            # mpi4py not installed
+            self.assertRaises(ImportError, choose_pool, 'mpi')
         else:
+            # mpi4py installed but only one process available
             self.assertRaises(ValueError, choose_pool, 'mpi')
 
```

Comment:
Thanks for fixing these. I'll try to run this on my not-mpi-installed machine and see if it works now.

Reply:
....and it works!